metals/src/main/scala/scala/meta/internal/metals/BloopInstall.scala

```diff
@@ -205,11 +205,25 @@ object BloopInstall {
   /**
    * Contents of metals.sbt file that is installed globally.
    */
-  private def globalMetalsSbt: String =
+  private def globalMetalsSbt: String = {
+    val resolvers =
+      if (BuildInfo.metalsVersion.endsWith("-SNAPSHOT")) {
```

Comment:
Can it ever be just "SNAPSHOT" like in the past?

Reply:
nope, it was nice while it lasted but caused troubles when the vscode plugin picked up ancient local builds from people who tried metals ages ago